cfme/tests/infrastructure/test_host.py

```diff
@@ -331,3 +334,59 @@ def test_hosts_not_displayed_several_times(appliance, provider, setup_provider):
     provider.wait_for_delete()
     provider.create()
     assert host_count == navigate_to(appliance.collections.hosts, "All").paginator.items_amount
+
+
+@pytest.fixture
+def setup_provider_min_hosts(request, provider, min_hosts=2):
+    num_hosts = len(provider.data.get('hosts', {}))
+    if num_hosts < min_hosts:
+        pytest.skip(f'Not enough hosts({num_hosts}) to run test. Need at least {min_hosts}')
+    """Function-scoped fixture to set up a provider"""
```

Comment:
**Optional** please use `#` is you want this comment in here

Reply:
I was thinking the same when I copied it. Though maybe you guys were doing it differently for some reason. I remember to use # for single line comments (as I usually do) going forward. Maybe I'll go back and fix this in the place I copied it from also in another PR.